docs/guides/cases/connect-http-device-via-gateway.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Connect a HTTP Device Via Gateway
+sidebar_position: 7
+---
+
+# Connect a HTTP Device Via Gateway
+
+## Introduction to LwM2M Gateway
+
+The LwM2M (Lightweight Machine to Machine) Gateway consists of two main components: an LwM2M client that connects to the LwM2M server and an HTTP client that connects to deviceShifu.  The LwM2M Gateway enables deviceShifu to adapt to the LwM2M protocol, handle requests from the LwM2M server, and push data to the cloud, fulfilling the two major functions required by the LwM2M protocol: the capability to pull data from the device and post data from the cloud to the device.
+
+## Pull and Deploy the Shifu Project
+
+Before pulling the project from GitHub, you need to download the relevant Git environment, which can be quickly installed using yum. After setting up Git, clone Shifu to your local machine. 
```

Comment:
we can exclude git install and shifu installation, or just leave a link to shifu install page is enough. it is too much for user to see these text as we assume they've already installed shifu to try these out.